docs/hooks/useAlert.md

```diff
@@ -1,13 +1,26 @@
-# `useAlert`
+# useAlert Hook
 
-`useAlert(message, options?) → { show }`
```

Comment:
Lines like this were kept in  'usage of the returned `show` function' and the 'useAlerts' page; are you interested in keeping/tossing all of them or is this a deliberate choice?